build/assets/custom-dictionary.txt

```diff
@@ -1,4 +1,4 @@
-personal_ws-1.1 en 367 utf-8
+personal_ws-1.1 en 456 utf-8
```

Comment:
It would be helpful, I think, to keep this file alphabetized/sorted (except for the first line)